CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Unreleased
 - BREAKING: Updated to `zbus` 2.0. This changes error types and public path fields.
 - BREAKING: `Error::Crypto` now contains a `&'static str` instead of a `String`.
 
+[2.0.2]
+- Increased minimum `zbus` version to 1.9.2, in order to increase the minimum version of the transitive dependency `nix` to at least 0.20.2, which is the first `nix` release to contain the fix for the security vulnerability described at https://rustsec.org/advisories/RUSTSEC-2021-0119 . A known issue with this version of `nix` is that it places an upper bound on the version of the `bitflags` dependency; if you are depending on `bitflags`, you may need to downgrade your `bitflags` version in order to upgrade to this version of `secret-service`, which you are encouraged to do in order to ensure that you are not exposed to the aforementioned `nix` vulnerability. In the long term, this will be fixed by upgrading `secret-service` to use a newer version of `zbus`, which itself depends on versions of `nix` which no longer have this restriction on `bitflags`.
```

Comment:
I believe that the dependency causing trouble is `enumflags2`, not `bitflags` (even though they provide very similar functionality, AFAICT).

Reply:
Looking at https://crates.io/crates/nix/0.20.2/dependencies, I see bitflags >=1.1.0,<1.3.0. I can see that zbus 1.9.2 has an enumflags2 dependency, but according to https://crates.io/crates/zbus/1.9.2/dependencies it's a simple ^0.6.4 version specifier with no problematic upper bound.

Reply:
See also https://github.com/hwchen/secret-service-rs/issues/38#issuecomment-1169454826